Makefile

```diff
@@ -127,7 +127,7 @@ gas: $(GAS)
 # convenience targets
 all: components tethers isos vic-machine vic-ui
 tools: $(GOIMPORTS) $(GVT) $(GOLINT) $(SWAGGER) $(GAS) goversion
-check: goversion goimports govet golint copyright whitespace gas
+check: goversion goimports govet lintcode copyright whitespace gas
```

Comment:
You mean if you name 'lintcode' to 'golint'?  Looks like you'd need to remove line 123 `golint: $(GOLINT)`.  Not sure why we have those set of targets is the first place (lines 122-125).  The `gas` target is overridden later too.  For example, 'make check' properly runs 'golint' again with the change below.  I also don't understand what `$(go-lint)` as a target was supposed to do as it isn't defined anywhere.

```diff
iff --git a/Makefile b/Makefile
index fb0a1fd..bcd1524 100644
--- a/Makefile
+++ b/Makefile
@@ -120,7 +120,6 @@ vic-ui: $(vic-ui-linux) $(vic-ui-windows) $(vic-ui-darwin)
 vic-dns: $(vic-dns-linux) $(vic-dns-windows) $(vic-dns-darwin)

 swagger: $(SWAGGER)
-golint: $(GOLINT)
 goimports: $(GOIMPORTS)
 gas: $(GAS)

@@ -175,7 +174,7 @@ whitespace:
 # exit 1 if golint complains about anything other than comments
 golintf = $(GOLINT) $(1) | sh -c "! grep -v 'should have comment'" | sh -c "! grep -v 'comment on exported'"

-$(go-lint): $(GOLINT)
+golint: $(GOLINT)
        @echo checking go lint...
        @$(call golintf,github.com/vmware/vic/cmd/...)
        @$(call golintf,github.com/vmware/vic/pkg/...)
```